src/controllers/alphabetical-search/search.controller.ts

```diff
@@ -32,13 +32,16 @@ const route = async (req: Request, res: Response) => {
         let showPrevLink;
         let showNextLink;
         let slicedCompanyResource;
+        let prevUrlSearchTerm;
+        let nextUrlSearchTerm;
 
         try {
             const companyResource: CompaniesResource =
                 await getCompanies(API_KEY, encodedCompanyName, cookies.get(SEARCH_WEB_COOKIE_NAME));
 
             const topHit: string = companyResource.topHit;
             const lastIndexPosition = companyResource.results.length - 1;
+            const showLinks = showPrevNextLinks(companyResource.results, topHit);
```

Comment:
`const [showPrevLink, showNextLink] = showPrevNextLinks(companyResource.results, topHit);`

You can then remove lines 35, 36, 91, 92